packages/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -56,7 +57,7 @@ describe('plugin-device', () => {
           device.checkNetworkReachability = sinon.spy();
         });
 
-        describe('when the \'intranetInactivityCheckUrl\' changes', () => {
```

Comment:
Why did we even write like this? hahaha. Glad you changed it

Reply:
Lint changed it, but yes it seems better this way.